pkg/operator/api/api.go

```diff
@@ -207,7 +207,7 @@ func (c *Cluster) apiContainer() v1.Container {
 		},
 		Env: []v1.EnvVar{
 			{Name: "ROOK_VERSION_TAG", Value: c.Version},
-			k8sutil.NamespaceEnvVar(),
+			{Name: "ROOK_NAMESPACE", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.namespace"}}},
```

Comment:
im suprised `ROOK_VERSION_TAG` and `ROOK_NAMESPACE` aren't consts. should they be?

Reply:
I tried to make it but this is part of a bigger change. The rook-api search for all env vars that start with "ROOK_". https://github.com/rook/rook/blob/master/cmd/rook/api.go#L49

I wanted to create constants so that it is reusable but i didnt want to have the prefix "ROOK_".